pyscriptjs/src/main.ts

```diff
@@ -387,4 +388,5 @@ const globalApp = new PyScriptApp();
 globalApp.main();
 
 export { version };
-export const runtime = globalApp.runtime;
+export const interpreter = globalApp.interpreter;
+export const runtime = globalApp.interpreter;
```

Comment:
The line `export const runtime = globalApp.interpreter;` is present for? Backward compatibility?
If it needs to be eventually removed, maybe we can add a comment here so that we don't forget in the next to next release.